tests/kwargs/test_multipart_data.py

```diff
@@ -188,9 +188,9 @@ def test_multipart_request_multiple_files_with_headers(tmpdir: Any) -> None:
                 "content": "<file2 content>",
                 "content_type": "text/plain",
                 "headers": [
-                    ["Content-Disposition", 'form-data; name="test2"; filename="test2.txt"'],
+                    ["Content-Disposition".lower(), 'form-data; name="test2"; filename="test2.txt"'],
                     ["x-custom", "f2"],
-                    ["Content-Type", "text/plain"],
+                    ["Content-Type".lower(), "text/plain"],
```

Comment:
```suggestion
                    ["content-type", "text/plain"],
```

Reply:
Fixed both in a force pushed change to keep commit message clean